db/models/comments.model.js

```diff
@@ -0,0 +1,125 @@
+"use strict";
+const { getValueToLowerCase } = require("../../helpers/db/getters");
+const { Model, DataTypes } = require("sequelize");
+
+class Comments extends Model {
+  static init(sequelize) {
+    super.init(
+      {
+        id: {
+          type: DataTypes.INTEGER,
+          autoIncrement: true,
+          primaryKey: true,
+          unique: true,
+        },
+        comment: {
+          type: DataTypes.STRING(255),
+          allowNull: false,
+        },
+        hidden: {
+          type: DataTypes.BOOLEAN,
+          defaultValue: true,
+        },
+        type: {
+          type: DataTypes.STRING,
+          allowNull: false,
+        },
+        issueId: {
+          type: DataTypes.INTEGER,
+          allowNull: false,
+          references: {
+            model: "issues",
+            key: "id",
+          },
+        },
+        proposalId: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+          references: {
+            model: "merge_proposals",
+            key: "id",
+          },
+        },
+        deliverableId: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+          references: {
+            model: "pull_requests",
+            key: "id",
+          },
+        },
+        userId: {
+          type: DataTypes.INTEGER,
+          allowNull: false,
+          references: {
+            model: "users",
+            key: "id",
+          },
+        },
+        userAddress: {
```

Comment:
Why Storing the adress if we already have the connection to the user tables?

Reply:
I just followed what was in the history of the card in jira